File: src/main/resources/static/Modules/Jasper/RepParams.js
/**
 * Объект, который передается построителю отчетов
 */
export class RepParams {
    /**
     * @param jrxml - имя отчета (без расширения)
     * @param beans - массив парамтров
     */
    constructor(jrxml, params) {
        this.jrxml=jrxml;
        this.params=params;
    }
}

File: src/main/java/org/programs/Accounting/Jasper/Models/RepParams.java
package org.kaznalnrprograms.Accounting.Jasper.Models;

public class RepParams {
    private String jrxml;
    private String postFix;

    private RepBean[] params;
    public String getJrxml()                {        return jrxml;    }
    public String getPostFix()              {        return postFix;    }
    public void setPostFix(String postFix)  {        this.postFix = postFix;    }
    public void setJrxml(String jrxml)      {        this.jrxml = jrxml;   }
    public RepBean[] getParams()            {        return params;    }
    public void setParams(RepBean[] params) {        this.params = params;    }
}

